utils/resource-monitor/graph_pidstat.py

```diff
@@ -0,0 +1,215 @@
+'''
+Graphs the performance of the batch job with HAPI as the source 
+
+This module runs a batch job with HAPI as the source and produces a graph and csv files of the performance of 
```

Comment:
Please break long lines and in general take a look at the [style guide](https://google.github.io/styleguide/pyguide.html#s3.2-line-length) (I have not commented on all style issues). You can also run the [black](https://github.com/psf/black) formatter as that is probably what we will use for automatic formatting.